src/core/stdlib.js

```diff
@@ -260,6 +260,12 @@ foam.LIB({
       if ( ! a || ! b ) return false;
       if ( a.equals ) return a.equals(b);
       return a == b;
+    },
+    function compare(a, b) {
+      if ( a.compareTo ) return a.compareTo(b);
```

Comment:
Did we add compareTo to the primitive types?  And if so, wont' this convert numbers into Numbers and end up being slow?